extensions/2.0/Vendor/EXT_mesh_features/schema/class.property.schema.json

```diff
@@ -15,41 +21,88 @@
       "description": "The description of the property."
     },
     "type": {
-      "type": "string",
-      "enum": [
-        "SINGLE",
-        "VEC2",
-        "VEC3",
-        "VEC4",
-        "MAT2",
-        "MAT3",
-        "MAT4",
-        "ARRAY"
-      ],
+      "description": "Element type represented by each property value. `VECN` is a vector with `N` components. `MATN` is an `N ⨉ N` matrix. `ARRAY` is fixed-length when `componentCount` is defined, and is variable-length otherwise.",
```

Comment:
```suggestion
      "description": "Element type represented by each property value. `VECN` is a vector of `N` numeric components. `MATN` is an `N ⨉ N` matrix of numeric components. `ARRAY` is fixed-length when `componentCount` is defined, and is variable-length otherwise.",
```